ansible/host_vars/node1.yml

```diff
@@ -1,2 +1,6 @@
 ---
 ansible_host: 172.16.238.12
+node_services:
+  HTTP.MyApp:
+    port: "{{ http_port }}"
+    path: "/"
```

Comment:
What we ideally want is to configure the service test for HTTP.MyApp in role apache2 and not with the node. Do think there is a way to do something like this?

Reply:
What I would like to do is something like this:

in `roles/apache2/defaults/main.yml`

```yaml
---
onms_services:
  HTTP.MyApp:
    port: "{{ http_port }}"
    path: "/"

```

in `roles/net-snmp/defaults/main.yml`

```
---
onms_services:
  HTTP.MyApp:
    port: "{{ http_port }}"
    path: "/"
```

The behavior is the last `onms_service` list wins and is not merged cause it's the same variable name.

Reply:
see https://github.com/opennms-forge/ansible-provisioning/pull/26#issuecomment-1641820665